Update examples and add one for cross account (#70)
DataDog/terraform-module-datadog-agentless-scanner: README.md

## Usage

To use this module in your Terraform configuration, add the following code in your existing terraform code:
```hcl
module "scanner_role" {
  source = "git::https://github.com/DataDog/terraform-module-datadog-agentless-scanner//modules/agentless-scanner-role"

  account_roles       = [module.delegate_role.role.arn]
  api_key_secret_arns = [module.agentless_scanner.api_key_secret_arn]
}

module "delegate_role" {
  source = "git::https://github.com/DataDog/terraform-module-datadog-agentless-scanner//modules/scanning-delegate-role"

  scanner_roles = [module.scanner_role.role.arn]
}

module "agentless_scanner" {
  source = "git::https://github.com/DataDog/terraform-module-datadog-agentless-scanner"

  api_key               = "YOUR API KEY"
  instance_profile_name = module.scanner_role.instance_profile.name
}
```

And run:
```

> ```

Author: Good catch ! 😅

> [!IMPORTANT]

> > We strongly recommend [pinning](https://developer.hashicorp.com/terraform/language/modules/sources#selecting-a-revision) the version of the module to keep repeatable deployment and to avoid unexpected changes.

Reviewer: ```suggestion
> Datadog strongly recommends [pinning](https://developer.hashicorp.com/terraform/language/modules/sources#selecting-a-revision) the version of the module to keep repeatable deployment and to avoid unexpected changes.
```

## Uninstall

> To uninstall, remove the agentless scanner module from your Terraform code. All resources associated with the agentless scanner will be deleted. Alternatively, if you've used a separate terraform state for this setup, you can directly execute `terraform destroy`.

Reviewer: ```suggestion
To uninstall, remove the Agentless scanner module from your Terraform code. Removing this module deletes all resources associated with the Agentless scanner. Alternatively, if you used a separate Terraform state for this setup, you can uninstall the Agentless scanner by executing `terraform destroy`.
```

> [!WARNING]

> > Please exercise caution when deleting Terraform resources. Review the plan carefully to ensure everything is in order.

Reviewer: ```suggestion
> Exercise caution when deleting Terraform resources. Review the plan carefully to ensure everything is in order.
```
